### ISSUE TITLE
Where are @OnMessage, @OnJsonObject,(server)addJsonObjectListener ?

### ISSUE THREAD

**Issue by bobpower67** [mrniko/netty-socketio#272]
Have copied jar to project (1.7.7) - mostly working (using OnEvent) but above not available. ( not using Maven )

**Comment by mrniko** [comment]
`OnMessage` and `OnJsonObject` has been dropped since socketio client 1.0

**Comment by stepjacky** [comment]
insteaded with what ?

**Comment by mrniko** [comment]
use `OnEvent`

**Comment by bobpower67** [comment]
Thanks for the quick response.

The Usage Notes on the main page include OnMessage, OnJsonObject etc. and as a newbie to web sockets I wasn't aware of the history - might be an idea to note them as pre version X in the Usage notes.

Thanks again,

Bob.

**Comment by pablojr** [comment]
@bobpower67 you're more than welcomed to collaborate with this project by, for example, updating the Usage notes.

Regards.

**Comment by bobpower67** [comment]
I've just spent 20 minutes of my lunch hr trying to figure out how to do that.... more confused than when I started. The GitHub WorkFlow doc is ... not much use either - it probably makes sense to someone who already understands it. If I understand correctly ; create a new fork, get a copy of that on my machine, edit the file, request someone review it, push it back ?? ( seems easier to let someone who's using it regularly do it, particularly given it's such a minor issue. )  Even so, how to do those things is not obvious from the GitHub online ui or desktop. Plainly this is something for another day.....possibly a whole weekend.
